share/dictionary/radius/dictionary.pfsense

```diff
@@ -1,11 +1,11 @@
 # -*- text -*-
-##############################################################################
+# Copyright (C) 2015 The FreeRADIUS Server project and contributors
```

Comment:
That should really be 2019...

Reply:
and you probably want to $INCLUDE the dictionary in the top level dictionary file